Sources/GoogleAI/GenerativeModel.swift

```diff
@@ -36,6 +36,8 @@ public final class GenerativeModel {
   /// The safety settings to be used for prompts.
   let safetySettings: [SafetySetting]?
 
+  let tools: [Tool]?
```

Comment:
Comment please. What is tools?

Reply:
Oh, I see https://ai.google.dev/api/rest/v1beta/Tool

Reply:
On this note, I also wonder if we could just accept `[FunctionDeclaration]` as a parameter instead of `[Tool]`.